cms/swap-rotating-banner.json

```diff
@@ -74,18 +46,12 @@
       "marginedImageAlt": "Diagramm, das vor einem Hintergrund aus Punkten exponentiell nach rechts oben zunimmt"
     },
     "en": {
-      "milkHeader": "Got Milk?",
```

Comment:
This only removed these for "en". Want me to jump in and remove for all localizations?

Reply:
Is there an automated way to do this?

Reply:
Not that I know of, or rather, I don't know of a way that's worth the effort of setting up given how infrequently we change this.